content/publication_resources/bibtex/lahti.bib

```diff
@@ -17,9 +17,12 @@ @Article{Jeyaram2025
   author =	 {Kumaraswamy Jeyaram and Leo Lahti and Sebastian Tims and Hans Heilig and Antonie van Gelder and Willem de Vos and Hauke Smidt and and Erwin Zoetendal},
   title =	 {Fermented foods affect the seasonal stability of gut bacteria in an Indian rural population},
```

Comment:
put Indian in curly brackets "{Indian}" so that the capitalization is kept in rendering (otherwise sentences are rendered to lowercase often)